docs/index.rst

```diff
@@ -61,3 +61,30 @@ Features
     how-to
     how-it-works
     changelog
+
+Known limitations
+-----------------
+pytest-xdist have some limitation that may be supported in pytest but can't be supported in pytest-xdist
+
+Order and amount of test must be constant
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Is is not possible to have test that differs in order or their amount across threads/execution.
```

Comment:
```suggestion
Is is not possible to have tests that differ in order or their amount across workers.
```

Reply:
this realy reads better 👍 good sugestion